File: android/app/src/main/java/br/dev/ci/MobManager/client/model/PhoneNumber.java
package br.dev.ci.MobManager.client.model;

public class PhoneNumber {
    String number;
    Device device;
    Integer subscriptionId;
    String carrierName;

    public String getNumber() {
        return number;
    }

    public void setNumber(String number) {
        this.number = number;
    }

    public Device getDevice() {
        return device;
    }

    public void setDevice(Device device) {
        this.device = device;
    }

    public String getCarrierName() {
        return carrierName;
    }

    public void setCarrierName(String carrierName) {
        this.carrierName = carrierName;
    }

    public Integer getSubscriptionId() {
        return subscriptionId;
    }

    public void setSubscriptionId(Integer subscriptionId) {
        this.subscriptionId = subscriptionId;
    }
}

File: client/projects/portal/api/src/lib/models/phone-number.ts
/* tslint:disable */
/* eslint-disable */
export interface IPhoneNumber {
  carrierName: string;
  device: {
};
  subscriptionId: number;
}

export class PhoneNumber implements IPhoneNumber {
  static typeName = 'PhoneNumber';
  private _carrierName: string;
  public get carrierName(): string{ return this._carrierName; }
  public set carrierName(value: string){
    if(this._carrierName === value) return;
    this._carrierName = value;
  }
  private _device: {
};
  public get device(): {
}{ return this._device; }
  public set device(value: {
}){
    if(this._device === value) return;
    this._device = value;
  }
  private _subscriptionId: number;
  public get subscriptionId(): number{ return this._subscriptionId; }
  public set subscriptionId(value: number){
    if(this._subscriptionId === value) return;
    this._subscriptionId = value;
  }
  toJSON(): IPhoneNumber{
    return {
          carrierName:this._carrierName,
          device:this._device,
          subscriptionId:this._subscriptionId,
    }
  }
}
